utils/gfw-meta.js

```diff
@@ -43,35 +43,23 @@ export default async function getGfwMeta() {
   };
 }
 
-export async function handleGladMeta(params) {
-  let GLAD;
-  if (isEmpty(params?.GFW_META?.datasets)) {
-    const meta = await getGfwMeta();
-    GLAD = meta?.datasets?.GLAD;
-  } else {
-    GLAD = params?.GFW_META?.datasets?.GLAD;
-  }
-  return GLAD;
-}
+export async function handleGfwParamsMeta(params) {
+  const paramsMeta = {
+    GLAD: null,
+    VIIRS: null,
+    INTEGRATED: null,
+  };
 
-export async function handleViirsMeta(params) {
-  let VIIRS;
   if (isEmpty(params?.GFW_META?.datasets)) {
     const meta = await getGfwMeta();
-    VIIRS = meta?.datasets?.VIIRS;
+    Object.keys(paramsMeta).forEach((key) => {
```

Comment:
I think we can avoid this forEach since the key you are using in `paramsMeta` and the key returned by `getGfwMeta` method is the same. You just need to return `meta.datasets` and get the right meta using destruct. 

```
export async function handleGfwParamsMeta(params) {
  const isMetaParamsEmpty = isEmpty(params.GFW_META.datasets)
  let gfwMetaParams;

  if (isMetaParamsEmpty) {
    gfwMetaParams = await getGfwMeta();
    
    return gfwMetaParams?.datasets
  }

  return params?.GFW_META?.datasets
}
```